test/e2e/common/runtime.go

```diff
@@ -200,7 +200,9 @@ while true; do sleep 1; done
 					phase:   v1.PodSucceeded,
 					message: Equal("OK"),
 				},
-			} {
+			}
+			for i := range testCases {
+				testCase := testCases[i]
```

Comment:
How about changing here with a single line like

```
for _, testCase := range testCases {
```
as line 77?

Reply:
BTW does line 253 in this test file also have the same issue?

```
253                         for _, testCase := range []struct {
254                                 description string
255                                 image       string
256                                 secret      bool
257                                 phase       v1.PodPhase
258                                 waiting     bool
259                         }{
260                                 {
```

Reply:
For your comment 1, I accepted you suggestion;

For you comment 2, the test in line 253 does not have the same issue, because inside the for loop, the line 298 redefine `testCase`:
`> 298				testCase := testCase`